openmc/mgxs/mgxs.py

```diff
@@ -1547,7 +1546,7 @@ def tallies(self):
         if self._tallies is None:
 
             # Create a list of scores for each Tally to be created
-            scores = ['flux', 'total', 'scatter-P1']
+            scores = ['flux', 'total', 'scatter-1']
```

Comment:
Oh wait, actually I was confused and thought you were commenting on the correction applied to `NuScatterMatrixXS`. I'm not so sure about `TransportXS`... I *think* that I agree with you in the case one is using `nu` in the scattering cross section. But in the case that one is not using `nu`, then we would *not* want the correction to account for `nu`. This requires a new setting in `TransportXS` to reflect the need for "nu-scatter-1" rather than "scatter-1." I've implemented this as a new option for the `correction` attribute as `"nuP0"` alongside the original `"P0"` setting. What do you think @nelsonag?